AutoRouteManager.php

```diff
@@ -50,11 +55,13 @@ class AutoRouteManager
     public function __construct(
         AdapterInterface $adapter,
         UriGeneratorInterface $uriGenerator,
-        DefunctRouteHandlerInterface $defunctRouteHandler
+        DefunctRouteHandlerInterface $defunctRouteHandler,
+        UriContextCollectionBuilder $collectionBuilder
```

Comment:
this is a BC break. It should at least be noted in the CHANGELOG, but it would be even better if we can avoid BC breaks.